server/index.ts

```diff
@@ -237,42 +254,53 @@ app.get("/*/contributors.txt", async (req, res) => {
   );
 });
 
-app.get(
-  [
-    "/:locale/curriculum/:slug([\\S\\/]+)/index.json",
-    "/:locale/curriculum/index.json",
-  ],
-  async (req, res) => {
-    const { slug = "" } = req.params;
-    const data = await findCurriculumPageBySlug(slug);
-    if (!data) {
-      return res.status(404).send("Nothing here 🤷‍♂️");
-    }
-    return res.json(data);
-  }
-);
+app.get("/*/runner.html", (_, res) => {
+  return res
+    .setHeader("Content-Security-Policy", PLAYGROUND_UNSAFE_CSP_VALUE)
+    .status(200)
+    .sendFile(path.join(STATIC_ROOT, "runner.html"));
+});
 
-app.get("/:locale/blog/index.json", async (_, res) => {
-  const posts = await allPostFrontmatter(
-    { includeUnpublished: true },
-    MEMOIZE_INVALIDATE
+if (CURRICULUM_ROOT) {
+  app.get(
+    [
+      "/:locale/curriculum/:slug([\\S\\/]+)/index.json",
+      "/:locale/curriculum/index.json",
+    ],
+    async (req, res) => {
+      const { slug = "" } = req.params;
+      const data = await findCurriculumPageBySlug(slug);
+      if (!data) {
+        return res.status(404).send("Nothing here 🤷‍♂️");
+      }
+      return res.json(data);
+    }
   );
-  return res.json({ hyData: { posts } });
-});
-app.get("/:locale/blog/author/:slug/:asset", async (req, res) => {
-  const { slug, asset } = req.params;
-  return send(
-    req,
-    path.resolve(
-      BLOG_ROOT,
-      "..",
-      "authors",
-      sanitizeFilename(slug),
-      sanitizeFilename(asset)
-    )
-  ).pipe(res);
-});
+} else {
+  console.warn("'CURRICULUM_ROOT' not set in .env file");
```

Comment:
we use the dev server in `content`, right? so will this cause unnecessary warnings for technical writers?